File: Project Euler/p6.py
# Sum square difference
def main():
    sum_square = 0
    square_sum = 0
    for i in range(1, 101):
        sum_square += i**2
        square_sum += i
    return (square_sum**2 - sum_square)

if __name__ == "__main__":
    print(main())

File: Project Euler/p6.cpp
// Sum square difference
#include <bits/stdc++.h>

using namespace std;

int main() {
    long sumSquare = 0; 
    long squareSum = 0;
    for (int i = 0; i < 101; i++) {
        sumSquare += pow(i, 2);
        squareSum += i;
    }
    long result = pow(squareSum, 2) - sumSquare; 
    cout << result << endl;
    return 0;
}
